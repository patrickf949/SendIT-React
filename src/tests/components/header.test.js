import React from "react";
import { shallow } from "enzyme";
import Header from "../../components/header";

describe("Dashboard", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
