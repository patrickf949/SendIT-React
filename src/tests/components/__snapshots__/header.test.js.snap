// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should render without crashing 1`] = `ShallowWrapper {}`;
